pkgs/development/libraries/libgpg-error/default.nix

```diff
@@ -24,6 +25,15 @@ in stdenv.mkDerivation (rec {
     sha256 = "sha256-/AfnD2xhX4xPWQqON6m43S4soelAj45gRZxnRSuSXiM=";
   };
 
+  patches = lib.optionals (with stdenv; buildPlatform != hostPlatform) [
```

Comment:
would you make a follow up pr for staging to make it unconditionally?

Reply:
Yes that's the plan

Reply:
There you go.